vue-components/src/components/Popover.vue

```diff
@@ -52,6 +53,20 @@ export default Vue.extend( {
 			type: Boolean,
 			default: true,
 		},
+		/**
+		 * Use this prop to set position of the popover in relation to the element that triggers it
+		 */
+		position: {
+			type: String,
+			validator( value: string ): boolean {
+				return value === null ||
+					[ 'top', 'right', 'bottom', 'left',
```

Comment:
those are repeated three times, once here and twice in the story. Let's extract them into an enum. Have a look at the `iconProps.ts` file for an example